feeds/wlan-ap/apc/src/src/ubus.c

```diff
@@ -159,10 +193,10 @@ ubus_init(void) {
 #endif
 	add_object(&apc_object);
 	notify_timer = tm_new_set(apc_send_notification, NULL,
-				  0, APC_NOTIFY_INTERVAL);
+				  0, APC_NOTIFY_CHECK);
 	if (notify_timer) {
-		printf("APC Start notify timer\n");
-		tm_start(notify_timer, APC_NOTIFY_INTERVAL);
+		printf("APC Start state check and notify timer\n");
+		tm_start(notify_timer, 1);
```

Comment:
Though the functionality would not change, you might want to use "APC_NOTIFY_CHECK" instead of hard-coded value 1 in tm_start().

Reply:
done